test/runtests.jl

```diff
@@ -69,16 +78,17 @@ if GROUP == "TTI_OP" || GROUP == "All"
     # TTI tests
     push!(Base.ARGS, "--tti")
     for t=devito
-        include(t)
+        @time include(t)
     end
 end
 
 # Anisotropic Acoustic tests with free surface
 if GROUP == "TTI_OP_FS" || GROUP == "All"
-    println("JUDI TTI operators with free surfacetests")
+    println("JUDI TTI operators with free surface tests")
+    deleteat!(devito, devito .== "test_gradient_twri.jl")
```

Comment:
Why is this necessary?

Reply:
Leftover from when `test_gradient_twri` was still run, removed.